### ISSUE TITLE
Rests are occasionally not parsed properly

### ISSUE THREAD

**Issue by elydpg** [alda-lang/alda-core#52]
Consider this segment:
```
thingNR = b-8 f b- f a- g f r
piano: thingNR*2
```

This will play `thingNR` twice as intended, however the final rest is clipped and it ends up lasting 14 counts instead of 16.

**Comment by elydpg** [comment]
It's also worth mentioning that 

```
thingNR = [b-8 f b- f a- g f r]
piano: thingNR*2
```

plays correctly.

**Comment by daveyarwood** [comment]
I found the issue -- there was an obscure bug in the parser that shows itself when you have a rest without a duration (`r`), followed by a newline, at the end of a variable definition. In that scenario, the parser was incorrectly parsing the newline first (I know, right?) and considering the rest to be outside of the variable definition.

I've got it fixed in 1.0.0-rc68. Thanks a lot for reporting this!
